internal/cfg/config.go

```diff
@@ -0,0 +1,199 @@
+package cfg
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	"github-file-sync/internal/log"
+)
+
+type Config struct {
+	RepositoryNames []string
+	FilesBindings   map[string]string
+
+	IsDryRun bool
+
+	GithubToken string
+	GithubURL   string
+
+	CommitMessage        string
+	PRTitle              string
+	FileSyncBranchRegexp string
+
+	Workspace      string // where the repository should be cloned
+	FileSourcePath string // where the source file are stored - set to current dir
+}
+
+// InitConfig based on env variables
```

Comment:
We follow linting based [here](https://github.com/FATMAP/avalanche-bulletin-service/blob/main/.golangci.yml) which includes using a period (`.`) at the end of comments. I will leave it up to SRE to decide but this is generally the linting and structure we follow for Go inside the mapping team
```suggestion
// InitConfig based on env variables.
```

I won't make a comment for each of these instances but just mention it once

Reply:
- I've copied/pasted the `.golang-ci.yml` file locally and applied most of the comment aside:
    - `fieldalignment` which I don't care about (I prefer structure readability over performances here).
    - `gomnd`: I don't want to declare static int here - it is less readable to me.
    - `gocritic`: telling me I don't document exported function when I actually do it, just not the way the rule expects it.